### PR TITLE
UI design of driver app updated.

### PR DESCRIPTION
**Pull request by yukthi14**
Splash screen, language screen and location permission screen changes are made.

### CODE REVIEW

File: lib/widgets/buttons/getStartedButton.dart [hunk 1]
```diff
@@ -23,26 +16,57 @@ class GetStartedButton extends StatefulWidget {
...
+    // TextButton(
+    //   // highlightColor: Colors.transparent,
+    //   onPressed: () {
+    //     widget.onTapNext!();
+    //   },
+    //   child: Container(
+    //     height: space_8,
+    //     decoration: BoxDecoration(
+    //       color: darkBlueColor,
+    //       borderRadius: BorderRadius.circular(radius_6),
```
Comment (Anik-Chakraborty) [inline lib/widgets/buttons/getStartedButton.dart:59]: Remove such commented codes as it has no uses

File: lib/screens/languageScreen.dart [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+import '../functions/trasnporterApis/runTransporterApiPost.dart';
+import '../language/localization_service.dart';
+import '../widgets/buttons/getStartedButton.dart';
+import '../widgets/loadingWidgets/bottomProgressBarIndicatorWidget.dart';
+import 'LoginScreens/loginScreen.dart';
+import 'navigationScreen.dart';
+
+class LanguageScreen extends StatefulWidget {
+  const LanguageScreen({super.key});
+
```
Comment (Anik-Chakraborty) [inline lib/screens/languageScreen.dart:18]: As you create this file for language then I don't find any need of previously exits languageSelectionScreen.dart file.
If there is no uses of languageSelectionScreen.dart and safely can be removed remove that file to ensure to dead file or code present in repo.

File: lib/screens/languageScreen.dart [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  getData() async {
+    bool? transporterApproved;
+    bool? companyApproved;
+    String? mobileNum;
+    bool? accountVerificationInProgress;
+    String? transporterLocation;
+    String? name;
+    String? companyName;
+
+    //transporterId = await runTransporterApiPost(
```
Comment (Anik-Chakraborty) [inline lib/screens/languageScreen.dart:50]: remove such commented code.

File: lib/screens/languageScreen.dart [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        transporterIdController.updateTransporterLocation(transporterLocation!);
+        transporterIdController.updateName(name!);
+        transporterIdController.updateCompanyName(companyName!);
+        print("transporterID is $transporterId");
+        setState(() {
+          _nextScreen = true;
+        });
+      }
+      //setState(() {
+      //_nextScreen=true;
```
Comment (Anik-Chakraborty) [inline lib/screens/languageScreen.dart:93]: do same for this one also

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**